romana-install/roles/provision/create-aws-hosts/templates/aws_cf_small_cluster.json

```diff
@@ -615,15 +118,10 @@ under the License.
             ]
           }
         }
-      },
-      "Metadata": {
-        "AWS::CloudFormation::Designer": {
-          "id": "4b80e302-c4a6-42ad-92e2-90b08b10f870"
-        }
       }
     },
-{% for node in cf_data.compute_hosts[:compute_nodes] %}
-    "{{ node }}": {
+{% for _ in range(compute_nodes) %}
+    "Compute{{ '%02d' | format(loop.index) }}": {
```

Comment:
They result in different expansions in this case, but generally yes, variables can be set inside blocks and reused.